test/e2e/fixtures.go

```diff
@@ -130,42 +129,42 @@ func setupTest(tb testing.TB) (*TestData, error) {
 }
 
 func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, error, bool) {
-	data := &TestData{}
+	// TODO: remove this hardcoding IPv4 setting after flow aggregator supports IPv6
 	isIPv6 := false
-	if err := data.setupLogDirectoryForTest(tb.Name()); err != nil {
-		tb.Errorf("Error creating logs directory '%s': %v", data.logsDirForTestCase, err)
+	if err := testData.setupLogDirectoryForTest(tb.Name()); err != nil {
+		tb.Errorf("Error creating logs directory '%s': %v", testData.logsDirForTestCase, err)
 		return nil, err, isIPv6
 	}
 	tb.Logf("Creating K8s clientset")
-	if err := data.createClient(); err != nil {
+	if err := testData.createClient(); err != nil {
 		return nil, err, isIPv6
 	}
```

Comment:
should not be needed - the clientset in `testdata` has already been configured in `testMain`

Reply:
Thanks, addressed.